frontend/src/views/camp/NavigationCamp.vue

```diff
@@ -40,7 +40,7 @@
   </v-app-bar>
   <v-bottom-navigation v-else app
                        fixed grow>
-    <v-btn>
+    <v-btn :to="{name: 'camp/material'}">
```

Comment:
```suggestion
    <v-btn :to="campRoute(camp(), 'material')">
```
Funktioniert sonst nicht richtig